po/ar.po

```diff
@@ -17,21 +17,29 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=6; plural=(n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 && "
 "n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5);\n"
-"X-Generator: Poedit 3.1.1\n"
+"X-Generator: Poedit 3.4.1\n"
 
 #: data/com.mattjakeman.ExtensionManager.desktop.in.in:3
 #: data/com.mattjakeman.ExtensionManager.metainfo.xml.in.in:6
 msgid "@app_title@"
 msgstr "@app_title@"
 
+#: data/com.mattjakeman.ExtensionManager.desktop.in.in:11
+msgid "Manage GNOME Shell Extensions"
+msgstr "إدارة إضافات جنوم شل"
+
+#. Translators: Do not translate
+#: data/com.mattjakeman.ExtensionManager.desktop.in.in:12
+msgid "extension;manager;"
+msgstr "إضافة; مدير;"
```

Comment:
This allows to search for the application by its name without translation, as normally the name of the application is translated. You can add more terms if you think it is necessary but always keeping the ones already in the original string.

```suggestion
#. Translators: Do not translate
#: data/com.mattjakeman.ExtensionManager.desktop.in.in:12
msgid "extension;manager;"
msgstr "extension;manager;إضافة; مدير;"
```